sdcm/results_analyze/__init__.py

```diff
@@ -222,15 +222,20 @@ def check_regression(self, test_id, data, is_gce=False):  # pylint: disable=too-
             build_id = ''
 
         last_events, events_summary = self.get_events(event_severity=[
-            Severity.CRITICAL.name, Severity.ERROR.name, Severity.WARNING.name])
-        reactor_stall_events, reactor_stall_events_summary = self.get_debug_events()
+            Severity.CRITICAL.name, Severity.ERROR.name])
+        debug_events, _ = self.get_debug_events()
+        reactor_stall_events = [stall for stall in debug_events if 'type=REACTOR_STALLED' in stall]
+        reactor_stall_events_summary = {Severity.DEBUG.name: len(reactor_stall_events)}
+        kernel_callstack_events = [stall for stall in debug_events if 'type=KERNEL_CALLSTACK' in stall]
+        kernel_callstack_events_summary = {Severity.DEBUG.name: len(kernel_callstack_events)}
         subject = f'Performance Regression Compare Results (latency during operations) -' \
                   f' {test_name} - {test_version} - {str(test_start_time)}'
         results = dict(
             events_summary=events_summary,
             last_events=last_events,
-            debug_events=reactor_stall_events,
-            debug_events_summary=reactor_stall_events_summary,
+            reactor_stall_events=reactor_stall_events[:300],  # limiting to 300 entries, avoiding oversize email
```

Comment:
i suggest to limit with 100.  i  think it will be enough to get picture about the reactor stalls. 
May be it is good to add warning if reactor stall number is higher than 100.

Reply:
reduced to 100